handsontable/src/plugins/contextMenu/menu/menuItemRenderer.js

```diff
@@ -3,6 +3,7 @@ import {
   isItemDisabled,
   isItemSelectionDisabled,
   isItemSeparator,
+  isItemCheckboxable
```

Comment:
I'm not sure if the naming is correct here @Skoczowianin 
fyi @jansiegel

Reply:
It probably could be improved. Maybe `isItemCheckable` would be more straightforward? `selectable` would make the most sense, but selection in this context is already taken.

Reply:
changed to isItemCheckable

Reply:
@jansiegel introduced `checkable` as the menu item property, which simplifies `isItemCheckable`